src/functions/sendCandidateResults/domain/personalisation.model.ts

```diff
@@ -0,0 +1,21 @@
+interface Personalisation {
+  'ref number' : string;
+  'test date' : string;
+  'test time' : string;
+  'first name' : string;
+  'cat dead' : string;
```

Comment:
Is this intentional? 🐱😅

Reply:
It's in the Template what's defined on Gov Notify (poor cat)

Reply:
Blame Peter Sen he requires we pass this into the template. Once we know what will be in the template it will be reworked.